miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -617,6 +617,14 @@ HotTarget.sample = (function() {
           HotUtils.makeColumnForInt('Slides Consumed', (show['Tissue Processing'] && config.targetSampleClass.alias == 'LCM Tube'),
               'slidesConsumed', HotUtils.validator.requiredPositiveInt),
 
+          // Tissue Processing: Single Cell columns
+          HotUtils.makeColumnForFloat('Initial Cell Conc.', (show['Tissue Processing'] && config.targetSampleClass.alias == 'Single Cell'),
+              'initialCellConcentration', false),
+          HotUtils.makeColumnForText('Digestion', (show['Tissue Processing'] && config.targetSampleClass.alias == 'Single Cell'),
```

Comment:
Since Digestion is not null, the `includes` will probably need to be updated to be shown when receiving stocks and aliquots too.